src/Resources/views/Order/Admin/Refund/Body/_bulkButtons.html.twig

```diff
@@ -0,0 +1,7 @@
+<div class="ui right aligned grid">
+    <div class="sixteen wide column">
+        <button data-refund-clear type="button" class="ui button" {{ disableButton }}>{{ 'sylius_refund.ui.clear_refunds'|trans }}</button>
+        <button data-refund-all type="button" class="ui button primary" {{ disableButton }}>{{ 'sylius_refund.ui.refund_all'|trans }}</button>
```

Comment:
I'm not sure if this makes sense and if it's not overengineering, but there could be one event here and these buttons would be added by blocks

Reply:
IMO good point, it opens possibility to more buttons 😄